res/index.html

```diff
@@ -4,7 +4,7 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-<meta charset="utf-8">
+<meta name="viewport" content="width=device-width" charset="utf-8" />  
```

Comment:
I believe we need 2 meta: one for the charset, the other one for the viewport. The meta charset is actually a shortcut for a longer `<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />`.

Now, maybe most browsers actually support what you propose, but I don't know how standard it is. Carefully reading the standard it looks like it could actually be supported, but I'd rather not count of it.

So please keep 2 meta tags :)

Also, about the actual `meta`, I usually use `<meta name="viewport" content="initial-scale=1" />`. I _think_ it has a better behavior on some iphones in some orientations (or when switching orientations).